### PR TITLE
exit if there is error while running the docker image in travis CI

### PR DESCRIPTION
**Pull request by Shekharrajak**
Related to this: https://github.com/numpy/numpy.org/pull/144#issuecomment-574793172

### CODE REVIEW

File: .scripts/install.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -euo pipefail
```
Comment (mattip) [inline .scripts/install.sh]: You might want to add `-x` which prints commands and their arguments as they are executed.

File: .scripts/build.sh [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
+set -euox pipefail
```
Comment (rgommers) [inline .scripts/build.sh]: Does this work? `pipefail` is connected to the `-o` option only. My bash isn't great, but I'm sure `set -euxo pipefail` works, not sure about this version

Comment (Shekharrajak) [reply .scripts/build.sh]: I think I need to create any build failure to verify it. in travis

**Comment by Shekharrajak** [comment]
It looks like it didn't work

**Comment by Shekharrajak** [comment]
related issue found: https://github.com/travis-ci/travis-ci/issues/8920

**Comment by Shekharrajak** [comment]
Still docker is returning non error code to travis CI. I have tried various way but still no use. It looks like I have to drop the idea of docker container and use the travis yaml file for the configuration.

**Comment by rgommers** [comment]
Yeah, Docker seems overkill - it's a few shell scripts that should be relatively straightforward, just `.travis.yml` should do it

**Comment by Shekharrajak** [comment]
Removed the docker part : https://github.com/numpy/numpy.org/pull/166

**Pull request closed without merging**